components/data-apps/index.md

```diff
@@ -83,20 +83,21 @@ To provide your app with environment variables or sensitive information like cre
 These secrets will be injected into the `secrets.toml` file upon deployment of the app. 
 [Read more about the Streamlit secrets](https://docs.streamlit.io/streamlit-community-cloud/get-started/deploy-an-app/connect-to-data-sources/secrets-management).
 
-### Default Secrets
-By default, the Keboola Streamlit Docker image includes two secrets that do not require explicit specification:
+## Access Storage from Data App
+By default, the Keboola Streamlit Docker image includes two environment variables, which can be easily used to access keboola storage from your application:
```

Comment:
Technicky ty ENVy nejsou soucasti toho image. Pokud bysme pustili image dodany userem (jednoho dne :slightly_smiling_face:), budou tam ty ENVy taky

```suggestion
By default, there are two environment variables, which can be easily used to access keboola storage from your application:
```

Reply:
aha, super díky. Takže jsi to změnil :-) takto je to tedy ok a můžeme approve?

Reply:
No ne, to je suggestion :slightly_smiling_face:

Reply:
Pokud s tim souhlasis, muzu to tam pridat (nebo muzes i sama)
![image](https://github.com/user-attachments/assets/6f23cec2-e456-433b-bbb8-3b0b3e3ad27b)

Reply:
@MonikaFeigler  asi musis data `Commit suggestion`, ja to pak rychle prolitnu

Reply:
commitla jsem to

Reply:
ajo, pardon, to jsem měla příjmout, díky moc oběma